src/ProjectTemplates/Web.ProjectTemplates/content/EmptyComponentsWebAssembly-CSharp/Client/Pages/Index.razor

```diff
@@ -0,0 +1,5 @@
+﻿@page "/"
+
+<h1>Hello, world!</h1>
+
+This is an empty Blazor WebAssembly template.
```

Comment:
I'd suggest not having this 5th line. The developer already knows it's the output from the empty Blazor WebAssembly template :)

"<h1>Hello world</h1>" is great for communicating "this is where your markup goes", so I don't think we need more than that.

Reply:
Let's update the Blazor Server empty template likewise in that case.

Reply:
Sounds good. I will update both templates to remove this line.